.cargo/config.toml

```diff
@@ -1,5 +1,5 @@
 [alias]
-xtask = "run --package xtask --bin xtask --quiet --"
+xtask = "run --package xtask --bin xtask --"
```

Comment:
Made `cargo xtask` no longer run quietly, as in my experience it made it less obvious whether a slowness issue was waiting for Cargo to rebuild `xtask` itself, or `xtask` being slow.

Reply:
Happy to see this tweak -- I also was frustrated that there was no feedback when `xtask` needed to be recompiled before it could run.